lib/nationbuilder/client.rb

```diff
@@ -39,13 +39,14 @@ def base_url
 
   RETRY_DELAY = 0.1 # seconds
 
-  def raw_call(path, method, body = {}, args = {})
+  def raw_call(path, method, body = {}, args = {}, opts = {})
     url = NationBuilder::URL.new(base_url).generate_url(path, args)
 
     request_args = {
       header: {
         'Accept' => 'application/json',
-        'Content-Type' => 'application/json'
+        'Content-Type' => 'application/json',
+        'X-Request-ID' => opts[:uniqueness_token]
```

Comment:
I've confirmed in the console that HTTPClient.send won't error with a header of nil:

`HTTPClient.send(:get, "http://google.com", {header: {'X-Request-ID' => nil}})`

Currently trying to ensure it won't cause the branch to succeed if `nil`